### ISSUE TITLE
Random failure in treeCopyHandler when running Detekt 1.20.0-RC1 Gradle tasks

### ISSUE THREAD

**Issue by davidburstrom** [detekt/detekt#4629]
I've only seen this happen once so far: I ran some standard Gradle `io.gitlab.arturbosch.detekt.Detekt` tasks in parallel and one of them threw an error quite similar to https://github.com/detekt/detekt/issues/4609. This also seems like a threading issue.

## Observed Behavior

```
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':detekt'.	
at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$1(ExecuteActionsTaskExecuter.java:147)	
•••
Caused by: org.gradle.api.GradleException: Duplicate registration for EP 'org.jetbrains.kotlin.com.intellij.treeCopyHandler': first in FakeIdForTests, second in FakeIdForTests	
at io.gitlab.arturbosch.detekt.invoke.DefaultCliInvoker.invokeCli(DetektInvoker.kt:65)	
at io.gitlab.arturbosch.detekt.Detekt.check(Detekt.kt:252)	
•••
Caused by: java.lang.reflect.InvocationTargetException: (No message provided)	
at io.gitlab.arturbosch.detekt.invoke.DefaultCliInvoker.invokeCli(DetektInvoker.kt:59)	
at io.gitlab.arturbosch.detekt.Detekt.check(Detekt.kt:252)	
•••
Caused by: java.lang.RuntimeException: Duplicate registration for EP 'org.jetbrains.kotlin.com.intellij.treeCopyHandler': first in FakeIdForTests, second in FakeIdForTests	
at org.jetbrains.kotlin.com.intellij.openapi.components.ComponentManager.createError(ComponentManager.java:167)	
at org.jetbrains.kotlin.com.intellij.openapi.extensions.impl.ExtensionsAreaImpl.checkThatPointNotDuplicated(ExtensionsAreaImpl.java:235)	
at org.jetbrains.kotlin.com.intellij.openapi.extensions.impl.ExtensionsAreaImpl.registerExtensionPoint(ExtensionsAreaImpl.java:240)	
at org.jetbrains.kotlin.com.intellij.openapi.extensions.impl.ExtensionsAreaImpl.doRegisterExtensionPoint(ExtensionsAreaImpl.java:210)	
at org.jetbrains.kotlin.com.intellij.openapi.extensions.impl.ExtensionsAreaImpl.doRegisterExtensionPoint(ExtensionsAreaImpl.java:189)	
at org.jetbrains.kotlin.com.intellij.openapi.extensions.impl.ExtensionsAreaImpl.doRegisterExtensionPoint(ExtensionsAreaImpl.java:183)	
at org.jetbrains.kotlin.com.intellij.openapi.extensions.impl.ExtensionsAreaImpl.registerExtensionPoint(ExtensionsAreaImpl.java:160)	
at io.github.detekt.parser.DetektPomModel.<init>(DetektPomModel.kt:27)	
at io.github.detekt.parser.KotlinEnvironmentUtilsKt.createKotlinCoreEnvironment(KotlinEnvironmentUtils.kt:56)	
at io.gitlab.arturbosch.detekt.core.settings.EnvironmentFacade$environment$2.invoke(EnvironmentAware.kt:38)	
at io.gitlab.arturbosch.detekt.core.settings.EnvironmentFacade$environment$2.invoke(EnvironmentAware.kt:31)	
at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)	
at io.gitlab.arturbosch.detekt.core.settings.EnvironmentFacade.getEnvironment(EnvironmentAware.kt:31)	
at io.gitlab.arturbosch.detekt.core.ProcessingSettings.getEnvironment(ProcessingSettings.kt)	
at io.gitlab.arturbosch.detekt.core.KtTreeCompiler.<init>(KtTreeCompiler.kt:14)	
at io.gitlab.arturbosch.detekt.core.tooling.ParsingStrategyKt$inputPathsToKtFiles$1.invoke(ParsingStrategy.kt:24)	
at io.gitlab.arturbosch.detekt.core.tooling.ParsingStrategyKt$inputPathsToKtFiles$1.invoke(ParsingStrategy.kt:23)	
at io.gitlab.arturbosch.detekt.core.tooling.Lifecycle$analyze$filesToAnalyze$1.invoke(Lifecycle.kt:38)	
at io.gitlab.arturbosch.detekt.core.tooling.Lifecycle$analyze$filesToAnalyze$1.invoke(Lifecycle.kt:38)	
at io.gitlab.arturbosch.detekt.core.util.PerformanceMonitor.measure(PerformanceMonitor.kt:42)	
at io.gitlab.arturbosch.detekt.core.tooling.Lifecycle$DefaultImpls.measure(Lifecycle.kt:34)	
at io.gitlab.arturbosch.detekt.core.tooling.Lifecycle$DefaultImpls.analyze(Lifecycle.kt:38)	
at io.gitlab.arturbosch.detekt.core.tooling.DefaultLifecycle.analyze(Lifecycle.kt:61)	
at io.gitlab.arturbosch.detekt.core.tooling.AnalysisFacade$runAnalysis$1.invoke(AnalysisFacade.kt:48)	
at io.gitlab.arturbosch.detekt.core.tooling.AnalysisFacade$runAnalysis$1.invoke(AnalysisFacade.kt:47)	
at io.gitlab.arturbosch.detekt.core.tooling.ProcessingSpecSettingsBridgeKt.withSettings(ProcessingSpecSettingsBridge.kt:26)	
at io.gitlab.arturbosch.detekt.core.tooling.AnalysisFacade.runAnalysis$detekt_core(AnalysisFacade.kt:47)	
at io.gitlab.arturbosch.detekt.core.tooling.AnalysisFacade.run(AnalysisFacade.kt:25)	
at io.gitlab.arturbosch.detekt.cli.runners.Runner.call(Runner.kt:33)	
at io.gitlab.arturbosch.detekt.cli.runners.Runner.execute(Runner.kt:23)	
at io.gitlab.arturbosch.detekt.invoke.DefaultCliInvoker.invokeCli(DetektInvoker.kt:59)	
at io.gitlab.arturbosch.detekt.Detekt.check(Detekt.kt:252)	
at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:104)
•••
```

## Steps to Reproduce

N/A

## Your Environment
* Version of detekt used: 1.20.0-RC1
* Version of Gradle used (if applicable): 7.4.1
* Operating System and version: macOS 12.2.1

**Comment by davidburstrom** [comment]
I suspect this was also fixed by https://github.com/detekt/detekt/pull/4631

**Comment by BraisGabin** [comment]
Closing for now then. Please reopen if you see that it wasn't fixed yet.

**Comment by cortinico** [comment]
I'll prepare a RC2 tomorrow, so we can give it another try 👍

**Comment by davidburstrom** [comment]
I'd appreciate that :)

**Comment by cortinico** [comment]
> I'll prepare a RC2 tomorrow, so we can give it another try 👍

Just a heads up that I'm waiting for those PRs to land before baking a RC2:
- https://github.com/detekt/detekt/pull/4632
- https://github.com/detekt/detekt/pull/4646
- https://github.com/detekt/detekt/pull/4645

**Comment by davidburstrom** [comment]
As far as I can tell, the bug is fixed in 1.20.0-RC2.
